src/test/kotlin/io/omnitrade/client/OmnitradeClientTest.kt

```diff
@@ -0,0 +1,264 @@
+package io.omnitrade.client
+
+import com.google.gson.FieldNamingPolicy
+import com.google.gson.GsonBuilder
+import com.winterbe.expekt.expect
+import com.winterbe.expekt.should
+import io.omnitrade.adapters.*
+import io.omnitrade.auth.Auth
+import io.omnitrade.enumerator.order.OrderDepth
+import io.omnitrade.enumerator.order.OrderSide
+import io.omnitrade.enumerator.order.OrderState
+import io.omnitrade.enumerator.order.OrderType
+import io.omnitrade.exception.OmnitradeNoAuthKeysException
+import io.omnitrade.fixtures.JsonResponses
+import io.omnitrade.model.Account
+import io.omnitrade.model.Deposit
+import io.omnitrade.model.Market
+import io.omnitrade.model.Member
+import io.omnitrade.repository.OmnitradeRepository
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.kodein.di.Kodein
+import org.kodein.di.generic.bind
+import org.kodein.di.generic.factory
+import org.mockito.ArgumentMatchers.anyMap
+import org.mockito.ArgumentMatchers.anyString
+import org.mockito.BDDMockito.given
+import org.mockito.Mockito.mock
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.text.DateFormat
+import java.time.Instant
+import java.util.*
+import kotlin.test.assertFailsWith
+
+object OmnitradeClientTest : Spek({
+  describe("#OmnitradeClient") {
```

Comment:
Should we have a `#` before the name of a class in tests? I see that it's been used only for method names.

Reply:
We actually shouldn't 👀 
Just fixed that.